.github/workflows/examples_manually_trigger.yaml

```diff
@@ -0,0 +1,46 @@
+name: Run Examples
+
+on: workflow_dispatch
+
+env:
+  GO111MODULE: on
+  SDK_VERSION: "1.3.0"
+  OPM_VERSION: "1.15.2"
```

Comment:
The target `release-operator` does not require opm . The target `prepare-operatorhub-pr` requires opm binary

Reply:
@Avni-Sharma `release-operator` actually does depend on `opm` transitively from [`index-image`](https://github.com/redhat-developer/service-binding-operator/blob/master/Makefile#L252) through `release-operator`->`push-index-image`->`index-image`